types/lodash/tsconfig.json

```diff
@@ -18,6 +18,391 @@
     },
     "files": [
         "index.d.ts",
-        "lodash-tests.ts"
+        "lodash-tests.ts",
+        "fp/add.d.ts",
```

Comment:
I don't think we need all of these anymore. You might need to fix `generalte-fp.ts` so it stops adding all of these.